x-pack/plugins/lens/server/migrations.test.ts

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { migrations, RawLensSavedXYObject770 } from './migrations';
```

Comment:
Where is this `migrations` file? Can't find in on master https://github.com/elastic/kibana/tree/master/x-pack/plugins/lens/server